palisade-service/src/main/resources/application-static.yaml

```diff
@@ -25,8 +25,7 @@ spring:
 spring:
   profiles: static-config
 
-# Discoveryless static configuration (presumably over localhost for integration-tests)
-# i.e. map from what-i-call-a-service -> service-URI
+# No Discovery static configuration
 web:
   client:
     audit-service: http://localhost:8081
```

Comment:
potential port change to avoid port conflicts

Reply:
done